app/src/main/java/com/example/plantsapp/data/repository/RoomTasksRepository.kt

```diff
@@ -7,6 +7,7 @@ import com.example.plantsapp.domain.model.Task
 import com.example.plantsapp.domain.repository.TasksRepository
 import javax.inject.Inject
 
+// TODO rename dao
```

Comment:
It is not a problem to rename it here - it will not affect any functionality, so you can do it here